docs/cloud-soar/cloud-soar-bridge.md

```diff
@@ -142,7 +142,7 @@ An example of a configuration file would be:
         "SOAR_URL":"https://YOUR_DOMAIN/incmansuite_ng/api", 
         "SOAR_TOKEN":"YOUR_JWT_TOKEN",
         "SIEM_URL":"https://YOUR_CSE_URL/sec",
-        "ALIAS": "YOUR_ALIAS_NO_SPACES"
+        "ALIAS": "YOUR_ALIAS_NO_SPACES_LESS_THAN_20_CHARACTERS"
```

Comment:
If the parameter is supposed to be less than 20 characters, this is a bad example, because it is more than 40 characters long. `"ALIAS": "YOUR_ALIAS"` would suffice.

Reply:
@jpipkin1 The proposed change is okay, so long as it explicitly states in the documentation that the ALIAS must be less than 20 characters